next-frontend/package.json

```diff
@@ -9,6 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@tanstack/react-query": "^5.67.2",
+    "openapi-fetch": "^0.13.4",
```

Comment:
Alphabetical order not kept. This is going to break with dependency updates eventually, creating an unnecessary diff in the future

Reply:
yeah I noticed myself due to next crashing (I think I messed up the yarn lock in merge)